zebra-chain/Cargo.toml

```diff
@@ -87,6 +87,7 @@ rayon = "1.6.1"
 # ZF deps
 ed25519-zebra = "3.1.0"
 redjubjub = "0.5.0"
+reddsa = "0.3.0"
```

Comment:
```suggestion
reddsa = "0.4.0"
```

Reply:
This also requires an update to `Cargo.lock`, CI will fail without it, because we use it to check for duplicate dependencies.

Reply:
done in 3ea23326e